### PR TITLE
updated model with changes that came out in implementation

### PR DESCRIPTION
**Pull request by ptrelease**

### CODE REVIEW

File: source/documentation/data_dictionary/tables/_funding_applications.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-organisation: UUID <<FK>>                   Link to organisation table
-created_at: Timestamp
-updated_at: Timestamp
+##applications
+
+Table that contains details specific to an application.
+
+```
+id: UUID <<PK>>                         
+declaration: Boolean                        records that decalration has been accepted
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_funding_applications.md:7]:
```suggestion
declaration: Boolean                        records that declaration has been accepted
```

File: source/documentation/data_dictionary/tables/_funding_applications.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
+declaration_description: Text               records applicant's permission to use certain info.
+declaration_keep_informed: Boolean          records that we can inform applicant
+declaration_user_research: Boolean          Agreed to User Research
+project_reference_number: String            Salesforce project reference number
+salesforce_case_id: String                  Case ID in Salesforce
+salesforce_case_number: String              Case Number is Salesforce
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_funding_applications.md:13]:
```suggestion
salesforce_case_number: String              Case Number in Salesforce
```

File: source/documentation/data_dictionary/tables/_missions.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-Lookup table for organisation mission types.
-For example, "female led"
-
-```
-id: Integer <<PK>>
-organision_mission_id: UUID <<FK>>
-name: String                                
-created_at: Timestamp
-updated_at: Timestamp
+##missions
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_missions.md:1]:
```suggestion
## missions
```

File: source/documentation/data_dictionary/tables/_organisation_types.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
-## organisation type
-
-Describes the type of organisation.  Such as TBA
-
-```
-id: Integer **PK**
-name: String
-created_at: Timestamp
-updated_at: Timestamp
+## organisation types
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_organisation_types.md:1]:
```suggestion
## organisation_types
```

File: source/documentation/data_dictionary/tables/_organisation_types.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
-Describes the type of organisation.  Such as TBA
-
-```
-id: Integer **PK**
-name: String
-created_at: Timestamp
-updated_at: Timestamp
+## organisation types
+
+Describes the type of organisation.  Such as TBA
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_organisation_types.md:3]:
```suggestion
Describes the type of organisation, e.g. 'local authority'.
```

File: source/documentation/data_dictionary/tables/_organisations.md [hunk 6]
```diff
@@ -1,16 +1,18 @@
...
-charity_number_ni: String           charity number specific for Northern Ireland
-salesforce_account_id: String           
-address_link_id: UUID <<FK>>        link to address_link table
-created_at: Timestamp
-updated_at Timestamp
+## organisations
+
+Records details about an organisation.
+
+Current table is called organisation.  Some migration
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_organisations.md:5]: Safe to remove this part - the table is `organisations` on the database.

File: source/documentation/data_dictionary/tables/_organisations_addresses.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+##organisations_addresses
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_organisations_addresses.md:1]: We've used `singular_plural` on other tables - we should be consistent in our naming convention. I don't have a preference as to either `singular_plural` or `plural_plural`.

File: source/documentation/data_dictionary/tables/_persons.md [hunk 8]
```diff
@@ -1,15 +1,14 @@
...
-```
-id: UUID <<PK>>
-name: String                        
-date_of_birth: Date
-email: String
-phone_number: String
-address_link_id: UUID <<FK>>        See address_link table.  A person can have more than one address.
-created_at: Timestamp
-updated_at: Timestamp
+##persons
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_persons.md:1]:
```suggestion
## persons
```

File: source/documentation/data_dictionary/tables/_persons_addresses.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+##persons_addresses
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_persons_addresses.md:1]:
```suggestion
## persons_addresses
```

File: source/documentation/data_dictionary/tables/_relationship_types.md [hunk 10]
```diff
@@ -1,10 +1,10 @@
-## relationship type
-
-Records the different kinds of relationship that might 
-exist between an application and person. For instance, there
-may be a person on an application that is a legal signatory.
-
-```
-  id: integer <<PK>>
-  name: String          Contains the relationship name
+## relationship types
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_relationship_types.md:1]:
```suggestion
## relationship_types
```

File: source/documentation/data_dictionary/tables/_small_sff_grant_programmes.md [hunk 11]
```diff
@@ -1,21 +1,24 @@
...
-description: Text                   Q) Describe your idea?
-start_date: Date                    Date the project starts       
-end_date: Date                      Date the project ends
-is_partnership: Boolean             Q) Are you applying on behalf of a partnership?
-partnership_details: Text           Q) Please tell us the organisation(s) you will be delivering your project in partnership...
-permission_type: id                 Q) Do you need permission from anyone else to do your project?
-permission_description: Text        Q) Please provide more information about who you need permission from to do your project.
-created_at: Timestamp               
-updated_at: Timestamp
+## small_sff_grant_programmes
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_small_sff_grant_programmes.md:1]: I think if we have the convention as `grant_programme_<name>`, then it's easier to immediately identify that the table is related to a grant programme. I know it'll break the plural naming convention, but I'm happy to do that with these particular tables if you are. Having `<name>_grant_programmes` makes it sound like it references more than one grant programme.

File: source/documentation/data_dictionary/tables/_supporting_evidences.md [hunk 12]
```diff
@@ -1,11 +1,11 @@
...
-
-Records the evidence that supports an application
-
-```
-id: UUID <<PK>>                     
-application_id: UUID <<FK>>
-description: Text                
-created_at: Timestamp
-updated_at: Timestamp
+## supporting_evidences
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_supporting_evidences.md:1]: This is one where I'm happy to break the plural naming convention.

File: source/documentation/data_dictionary/tables/_volunteers.md [hunk 13]
```diff
@@ -1,12 +1,12 @@
...
-Records details of volunteers included in an application. 
-
-```
-id: UUID <<PK>>
-application_id: UUID <<FK>>
-hours: Integer                      hours that the volunteer works (per...)
-description: Text                   describes the work of the volunteer
-created_at: Timestamp
-updated_at: Timestamp
+##volunteers
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_volunteers.md:1]:
```suggestion
## volunteers
```

File: source/documentation/design/data_model.html.md.erb [hunk 14]
```diff
@@ -13,37 +13,39 @@ cases as quickly and easily as possible.
 <div class="mermaid">
```
Comment (stuartmccoll) [inline source/documentation/design/data_model.html.md.erb:13]: Placeholder comment - any changes in other files will need to be replicated in this one.

File: source/documentation/data_dictionary/tables/_small_sff_grant_programmes.md (hunk 11 above)
Comment (ptrelease) [reply source/documentation/data_dictionary/tables/_small_sff_grant_programmes.md]: Changed as discussed

**Pull request merged**